api/v1alpha1/gitopsservice_types.go

```diff
@@ -17,13 +17,19 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // GitopsServiceSpec defines the desired state of GitopsService
 type GitopsServiceSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// InfraNodeEnabled will add infra NodeSelector to all the default workloads of gitops operator
+	InfraNodeEnabled bool `json:"infraNodeEnabled,omitempty"`
```

Comment:
Can we change this from `InfraNodeEnabled` to `RunOnInfra` ?

Reply:
sure